client/Dockerfile.dev

```
@@ -6,6 +6,15 @@ FROM node:20-slim
 
 WORKDIR /app
 
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)